Svelte kit fixes

linuxuser586 opened the pull request:
This is a proposed fix for `process is not defined` as mentioned in issue #703.

pubkey commented:
I just released a new version. Can you check if this solves your problem?
Also please do not commit generated files in pull requests.

catrielmuller commented:
@pubkey , I need the package.json export change ( https://github.com/pubkey/broadcast-channel/pull/704/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R6 ) to resolve the issue with Webpack 5 

```
Module not found: Error: Exports field key should be relative path and start with "." (key: "node")
```

linuxuser586 commented:
@pubkey Yes, with a small change. I needed to change package.json to follow the specification. I changed it to the following:

```json
"exports": {
    ".": {
      "node": {
        "import": "./dist/esnode/index.js",
        "default": "./dist/es5node/index.js"
      },
      "default": {
        "import": "./dist/esbrowser/index.js",
        "default": "./dist/lib/index.es5.js"
      }
    },
    "./package.json": "./package.json"
  },
```
I changed browser to default as recommended in the documentation. 
https://nodejs.org/dist/latest-v14.x/docs/api/packages.html#packages_conditional_exports

> When using environment branches, always include a "default" condition where possible. Providing a "default" condition ensures that any unknown JS environments are able to use this universal implementation, which helps avoid these JS environments from having to pretend to be existing environments in order to support packages with conditional exports. For this reason, using "node" and "default" condition branches is usually preferable to using "node" and "browser" condition branches.
